HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -23,10 +23,9 @@ enum class RotationConvention
     ZYX_Intrinsic,
     XYZ_Extrinsic,
     XYZ_Intrinsic,
-    ZYX_Extrinsic,
-    NOF_ROT
+    ZYX_Extrinsic
 };
-constexpr size_t nofRotationConventions = static_cast<size_t>(RotationConvention::NOF_ROT);
+constexpr size_t nofRotationConventions = static_cast<size_t>(RotationConvention::ZYX_Extrinsic);
```

Comment:
Did you test this? Won't you now skip `ZYX_Extrinsic` when iterating?

Reply:
I've added + 1 to fix this. Not sure if there is a better way, but it works as it should now.
It seems there is no way to get the size of elements in an enum, except like we had with NOF_ROT which was used as a 'Count' from here:
https://stackoverflow.com/questions/14989274/is-it-possible-to-determine-the-number-of-elements-of-a-c-enum-class
The point is that we want this constexpr to be equal to the size of the enum (even if the enum is expanded). Any suggestions for a better method?

Reply:
There's indeed no way to get the count of elements in an enum. I think the `NOF_ROT` trick is ok, it's been so long since I last looked at this PR that I don't remember if we discussed whether/why to remove it. Do you?

Reply:
Because we get a warning that it is unused.